agent/acs/model/ecsacs/api.go

```diff
@@ -288,6 +288,8 @@ type Container struct {
 
 	Environment map[string]*string `locationName:"environment" type:"map"`
 
+	EnvironmentFiles []*EnvironmentFile `locationName:"environmentFiles" type:"list"`
```

Comment:
this file is autogenerated. you should edit `model/api/api-2.json` and then go generate this file. otherwise the next people generating api.go will remove the changes you added here

Reply:
lol i was thinking that was probably the case thanks